include/deal.II/base/array_view.h

```diff
@@ -98,6 +98,8 @@ class ArrayView
    */
   ArrayView();
 
+  ArrayView(value_type &element);
+
```

Comment:
@bangerth I have tried out you suggestion regarding automatic conversion. It works for `std::vector()` but not for `double` at the moment. Does anything speaks against to introduce a new constructor in `ArrayView` which takes a reference to a `value_type`?  Then the code below could look like `min_max_avg(my_value, result, mpi_communicator);`.

If nothing speaks against that, I would create a separate PR.

Reply:
I see no reason why we shouldn't have such a constructor. I like the idea!